src/components/ThemeProvider/Ant.tsx

```diff
@@ -22,6 +22,7 @@ import { ReactElement, useMemo } from 'react'
 import { DEFAULT_COLOR, DEFAULT_FONT_SIZE } from './utils/themeDefaultStyle'
 import ButtonTheme from '../Button/Button.theme'
 import DividerTheme from '../Divider/Divider.theme'
+import FeedbackMessageTheme from '../Message/Message.theme'
```

Comment:
Why are 2 nomenclatures ("Message" and "FeedbackMessage") used to refer to the same component? It creates confusion. I would always use "FeedbackMessage" (thus going to rename all files and occurrences) since that is what we are talking about, and "Message" is too generic as a name for this specific component

Reply:
There was an error on my side, the name should be `MessageTheme` as the path suggests.

I gave a thought on the naming, I decided to keep the `useFeedbackMessage` as the message to return a feedback (the component that pops up at the top of the page) that includes a `Message` (the component containing the text and - eventually - some extra content on the right side).

Reply:
However, I think "Message" is too generic for this component. A message is a string, whereas here we have a button containing, in addition to the string itself, an extra (tending to be a status, as we use it). If we were to add the "FormattedMessage" to the design system it would cause further confusion

Reply:
Using `FeedbackMessage` for the component as well.